src/javascripts/views/budgetView.js

```diff
@@ -12,6 +13,19 @@ class BudgetView {
         this.overlayElement.classList.add('overlay-active');
     }
 
+    closeBudgetPopup() {
+        this.budgetElement.classList.remove('budget-active');
+        this.overlayElement.classList.remove('overlay-active');
+    }
+
+    resetBudgetPopup() {
+        const inputs = this.budgetBodyElement.querySelectorAll('input');
+
+        inputs.forEach((input) => {
+            input.value = ' ';
```

Comment:
Have you try reset() for form?

Reply:
Yes, I have tried reset() for form @phuocduong-agilityio